src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/StageDetail.component.js

```diff
@@ -146,17 +147,20 @@ const StageDetailPlain = (props: Props) => {
                                 key={id}
                                 onClick={() => !row.pendingApiResponse && onEventClick(row.id)}
                                 ref={(tableCell) => {
-                                    if (tableCell && row.pendingApiResponse) {
-                                        tableCell.onmouseover = onMouseOver;
-                                        tableCell.onmouseout = onMouseOut;
-                                        ref.current = tableCell;
+                                    if (tableCell) {
```

Comment:
That's a fair question - the drawback with this approach is that the tooltip will only show up when you hover the text. In the current implementation it is sufficient to put the mouse anywhere within the table cell. There was a similar thing with some of the buttons I've refactored too, and whenever I encountered that I made sure the whole button rather than the text within it would trigger the tooltip.